src/addrman.cpp

```diff
@@ -31,19 +31,24 @@ int CAddrInfo::GetBucketPosition(const uint256 &nKey, bool fNew, int nBucket) co
 
 bool CAddrInfo::IsTerrible(int64_t nNow) const
 {
-    if (nLastTry && nLastTry >= nNow - 60) // never remove things tried in the last minute
+    /// never remove things tried in the last minute
```

Comment:
As pointed out, we also don't want these comments included, as they make the most sense in the context of the code. I'll add some screenshots of what they look like in Doxygen to this thread.